stand/lua/loader.conf.lua.5

```diff
@@ -29,7 +29,7 @@
 .Os
 .Sh NAME
 .Nm loader.conf.lua
-.Nd Lua-based system bootstrap configuration file
+.Nd system bootstrap Lua configuration information
```

Comment:
This one I did a little bit differently to match the style used in loader.conf(5)